client/src/Picks.tsx

```diff
@@ -1004,6 +1005,48 @@ function ForceBeatmapsetUpdate({ nomination }: { nomination: INomination }) {
   );
 }
 
+interface DescriptionDifferenceProps {
+  oldDescription: string;
+  newDescription: string;
+}
+
+function DescriptionDifference({
+  oldDescription,
+  newDescription,
+}: DescriptionDifferenceProps): JSX.Element {
+  if (oldDescription === newDescription) {
+    return <span>{newDescription}</span>;
+  }
+  const diffMatch = new diff_match_patch();
+  const differences = diffMatch.diff_main(oldDescription, newDescription, false);
+
+  // cleanup
+  diffMatch.diff_cleanupSemantic(differences);
+  //"color: #698542; background-color: #95AE86;"
+  // "color: #854242; background-color: #AE8686;"
+  return (
+    <>
+      {differences.map((difference, i) => {
```

Comment:
using `i` as key breaks if the descriptions passed ot this component change
(they never do, so this is "fine", but maybe warrants a comment or something in case this component is ever used elsewhere)

Reply:
unsure what else to use as the key tbh